test/Microsoft.AspNet.Razor.Runtime.Test/PlatformHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.Test
+{
+    public static class PlatformHelper
+    {
+        public static bool IsMono { get; } = Type.GetType("Mono.Runtime") != null;
```

Comment:
remove. general direction is to avoid asserting about lower-level `Exception` messages.